setup.py

```diff
@@ -131,7 +132,7 @@ def get_env_if_set(key, default: typing.Any = ""):
     TORCH_MAJOR = "0"
     TORCH_MINOR = "0"
 
-if torch_available and not torch.cuda.is_available():
+if torch_available and not get_accelerator().device_name() == 'cuda':
```

Comment:
Might be worth checking this on AMD devices as well since we want to ensure we don't break any compat there?  @jithunnair-amd thoughts?

Reply:
@loadams I don't think this should break any compat with ROCm, since we alias as `cuda` in the accelerator.